src/caches/sdirk_caches.jl

```diff
@@ -75,6 +75,17 @@ DiffEqBase.@def oopnlcachefields begin
   _nlsolve = oop_nlsolver(alg.nlsolve)
```

Comment:
This line is no longer needed together with https://github.com/JuliaDiffEq/StochasticDiffEq.jl/pull/119/files#diff-764be46c2a94bdca1a4b0bc276f043d4R45.